app/Http/Controllers/User/MyRequestController.php

```diff
@@ -81,6 +84,20 @@ public function update(Request $request, $id)
             $this->notification->store($data);
         }
 
+        if ($request->status == 'returning') {
+            $book_id = $this->bookUser->findId($id)->book_id;
+            $count_returnedBook = $this->bookUser->countReturned($book_id);
+            $find_Returned = $this->bookMeta->findReturned($book_id);
+            if (empty($find_Returned)) {
+                $a = $this->bookMeta->insertReturned([
+                    'key' => 'count_returned',
+                    'value' => $count_returnedBook->book_returned,
+                    'book_id' =>$book_id,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/User/MyRequestController.php</br>**Line**: 95</br>Expected 1 space between double arrow and "$book_id"; 0 found